Taskfile.yml

```diff
@@ -0,0 +1,53 @@
+version: '3'
+
+set: [pipefail]
+
+run: when_changed
+
+vars:
+  REPO_ROOT: "{{.ROOT_DIR}}"
+  BUILD_ROOT: "{{.ROOT_DIR}}/build"
+  DOCKER: '{{.DOCKER | default "docker"}}'
+
+includes:
+  metrics_server:
+    taskfile: ./src/metrics_server/Taskfile.yml
+    vars: {OUTPUT_BASE: '{{joinPath .BUILD_ROOT "metrics_server"}}'}
+
+  sentry_webhook:
+    taskfile: ./src/sentry_webhook/Taskfile.yml
+    vars: {OUTPUT_BASE: '{{joinPath .BUILD_ROOT "sentry_webhook"}}'}
+
+  shadowbox:
+    taskfile: ./src/shadowbox/Taskfile.yml
+    vars: {OUTPUT_BASE: '{{joinPath .BUILD_ROOT "shadowbox"}}'}
+
+tasks:
+  clean:
+    desc: Clean output files
+    cmds:
+      - rm -rf .task src/*/node_modules/ build/ node_modules/ third_party/shellcheck/download/ third_party/*/bin third_party/jsign/*.jar
```

Comment:
Do we need to remove the `./task` binary built by `npm install` as well?

Reply:
Makes sense. Done